apsuite/commisslib/meas_coupling_tune.py

```diff
@@ -138,17 +139,22 @@ def __init__(self, isonline=True):
             params=CouplingParams(), target=self._do_meas, isonline=isonline)
         self.qs_names = None
         self.apply_factor = 0
-        self.initial_strengths = None
         if self.isonline:
             self._create_devices()
+            self.initial_strengths = self.get_initial_strengths()
```

Comment:
I don't think it is a good idea to get the initial strengths here. I think you should get it when it is needed for the first time.